client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -1,58 +1,68 @@
 /**
  * External dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	ReactDomServer = require( 'react-dom/server' ),
-	ReduxProvider = require( 'react-redux' ).Provider,
-	React = require( 'react' ),
-	tinymce = require( 'tinymce/tinymce' ),
-	pick = require( 'lodash/pick' ),
-	assign = require( 'lodash/assign' ),
-	values = require( 'lodash/values' ),
-	debounce = require( 'lodash/debounce' ),
-	includes = require( 'lodash/includes' ),
-	i18n = require( 'i18n-calypso' ),
-	Shortcode = require( 'lib/shortcode' ),
-	closest = require( 'component-closest' );
+import ReactDom from 'react-dom';
+import ReactDomServer from 'react-dom/server';
+import { Provider as ReduxProvider } from 'react-redux';
+import React from 'react';
+import tinymce from 'tinymce/tinymce';
+import { assign, debounce, find, findLast, pick, values } from 'lodash';
+import i18n from 'i18n-calypso';
+import Shortcode from 'lib/shortcode';
+import closest from 'component-closest';
 
 /**
  * Internal dependencies
  */
-var sites = require( 'lib/sites-list' )(),
-	PostActions = require( 'lib/posts/actions' ),
-	PostEditStore = require( 'lib/posts/post-edit-store' ),
-	MediaConstants = require( 'lib/media/constants' ),
-	MediaActions = require( 'lib/media/actions' ),
-	MediaUtils = require( 'lib/media/utils' ),
-	MediaSerialization = require( 'lib/media-serialization' ),
-	MediaMarkup = require( 'post-editor/media-modal/markup' ),
-	MediaModalViews = require( 'post-editor/media-modal/constants' ).Views,
-	MediaStore = require( 'lib/media/store' ),
-	MediaLibrarySelectedData = require( 'components/data/media-library-selected-data' ),
-	EditorMediaModal = require( 'post-editor/media-modal' ),
-	notices = require( 'notices' ),
-	TinyMCEDropZone = require( './drop-zone' ),
-	restrictSize = require( './restrict-size' ).default,
-	advanced = require( './advanced' ),
-	Gridicon = require( 'components/gridicon' ),
-	config = require( 'config' );
+import SiteListFactory from 'lib/sites-list';
+const sites = SiteListFactory();
```

Comment:
Could we move this down to below the "Module variables" block to keep these exclusively for imports?